### ISSUE TITLE
More details about finetuning on new robots & tasks

### ISSUE THREAD

**Issue by meng-zha** [openvla/openvla#12]
Thank you for the exciting work!

Can you provide more details about finetuning on new robots & tasks?
I tried to finetune the pretrained openvla model on only one new task of a new robot but I did not obtain a promising result. Maybe more details can help such as:

- learing_rate,
- max_epochs,
- batch size,
- the size of training dataset,
- image augmentation,
- train more epochs to overfit or not?

**Comment by kpertsch** [comment]
You can follow the defaults in the `finetune.py` script -- Moo Jin recently updated the default learning rate there so you could just try re-running. Some things that worked well in the past:
- training dataset size: ~100 episodes
- image augmentation: yes -- it's turned on by default in the finetuning script
- max_epochs: we usually train the model to ~95+% action token accuracy, which often takes a few epochs to get there

Also be sure to apply center crop augmentation at inference time when rolling out the model if you trained with data augmentation. Below is an example of how we compute center crop at inference time to match our training defaults:

```
temp_image = np.array(image)  # (H, W, C)
crop_scale = 0.9
sqrt_crop_scale = math.sqrt(crop_scale)
temp_image_cropped = apply_center_crop(
    temp_image, t_h=int(sqrt_crop_scale * temp_image.shape[0]), t_w=int(sqrt_crop_scale * temp_image.shape[1])
)
temp_image = Image.fromarray(temp_image_cropped)
temp_image = temp_image.resize(image.size, Image.Resampling.BILINEAR)  # IMPORTANT: dlimp uses BILINEAR resize
image = temp_image
```
